examples/ota-provider-app/esp32/main/main.cpp

```diff
@@ -139,7 +139,8 @@ static void InitServer(intptr_t context)
     fclose(otaImageFile);
     otaImageFile = NULL;
 
-    chip::app::Clusters::OTAProvider::SetDelegate(kOtaProviderEndpoint, &otaProvider);
+    chip::app::Clusters::OTAProvider::ResetDelegate();
+    chip::app::Clusters::OTAProvider::ResetDelegate(kOtaProviderEndpoint, &otaProvider);
```

Comment:
It seems to be different from the darwin implementation.
ResetDelegate(), SetDelegate()

Reply:
typo. fixed, thanks